server/sonar-server/src/main/java/org/sonar/server/computation/task/projectanalysis/api/posttask/PostProjectAnalysisTasksExecutor.java

```diff
@@ -112,11 +113,15 @@ private static void executeTask(ProjectAnalysis projectAnalysis, PostProjectAnal
 
   private ProjectAnalysis createProjectAnalysis(CeTask.Status status) {
     Long analysisDate = getAnalysisDate();
+    Analysis analysis = null;
+    if (analysisDate != null) {
+      analysis = new Analysis("", analysisDate); // FIXME
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Take the required action to fix the issue indicated by this comment. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://next.sonarqube.com/sonarqube/coding_rules#rule_key=squid%3AS1134)